### DISCUSSION TITLE
My sidebar is broken, docfx:navrel and docfx:tocrel are wrong

### DISCUSSION THREAD

**Discussion by sliekens** [dotnet/docfx#9478] [Q&A]
Hi,

I'm having problems with my docfx setup: https://github.com/sliekens/gw2sdk/blob/main/docfx.json

For some weird reason, if I build and deploy this to GitHub pages, the metadata attributes are wrong.
https://sliekens.github.io/gw2sdk/guide/introduction.html

Expected:

```
<meta property="docfx:navrel" content="../toc.html">
<meta property="docfx:tocrel" content="toc.html">
````

Actual:
```
<meta property="docfx:navrel" content="../toc.json">
<meta property="docfx:tocrel" content="toc.json">
````

As a result, the sidebar doesn't load. Also, the code keeps trying to load the sidebar without ever giving up.

I tried to reproduce it on my machine, but it worked as expected. Does anyone know what is going on here?

Debug logs: 
[5_Build documentation site.txt](https://github.com/dotnet/docfx/files/13466760/5_Build.documentation.site.txt)

Thanks for any help.

**Comment by glb-cblin** [comment]
I reproduce and added details in #9479
